packages/manager/apps/container/src/container/legacy/server-sidebar/index.tsx

```diff
@@ -49,12 +49,12 @@ export default function ServerSidebarIndex() {
     } else {
       setIsAccountMenu(false);
     }
-  }, [universe, location]);
+  }, [universe, location, application]);
 
   if (universe === 'hub') {
     return <></>
   }
-  if (isAccountMenu) {
+  if (isAccountMenu || location.pathname.startsWith(`/iam`)) {
```

Comment:
IMO, the `useEffect` check if the current `application` correspond to `iam` and enable accountMenu for all `iam` routes (`*`).
Adding `application` in `useEffect`'s dependencies is not sufficient to fix the issue ?

Reply:
Hello, yes you're right, this part was only introduced to resolve the first issue in the ticket. But then Teddy found another linked problem and so I added the application in the useEffect and forgot to remove this check.

It's done now, thx.